examples/hana-scale-up-perf-optimized-azure.yaml

```
@@ -304,18 +304,23 @@ groups:
     description: "Check OS and package versions are supported and up-to-date"
     checks:
       - id: 2.2.1
-        description: "Check the base product"
+        description: "Check SLES4SAP is installed"
```

Maybe we should make this more generic, so that this could be in future parameterised to different values.
Something like:
```suggestion
        description: "Check a supported OS distribution is being used"
```